docs/setup.md

```diff
@@ -70,3 +70,29 @@ Run dev electron (to use the dev server you need to run this)
 ```shell
 yarn dev-electron
 ```
+
+## Working on the Shared Libraries and the Portal Extension
+
+The Portal Extension relies on packages provided by the shared libraries. The portal installs the necessary dependencies that are published on a remote registry. If you want to work on the portal in tandem with local, unpublished versions of the shared libraries, you will need to run the following commands:
+
+
+1. Install the Batch Utility CLI
+
+```shell
+cd ./util/build-util
+yarn global add $PWD
```

Comment:
Is $PWD needed here? Does `yarn global add .` work? I believe $PWD won't work in a Windows command prompt (but is fine in Powershell).

Reply:
Yeah, I thought of that too, but no, `yarn global add .` alas doesn't work. :(

Reply:
Fixed documentation.